packages/stencil/src/components/datacom-breadcrumb/usage.md

```diff
@@ -3,9 +3,14 @@
 Breadcrumb has one component 'datacom-breadcrumb'
 ### Properties - datacom-breadcrumb
 
--url for passing link of other page. Styled with 'dc- breadcrumb' class.
+-url for passing link of other page. 
```

Comment:
Having a hard time understanding what you're trying to say with these documentation updates here. Might want to revise your grammar here.